src/hooks/tradingView/useChartLines.ts

```diff
@@ -189,59 +222,187 @@ export const useChartLines = ({
     });
   }, [stringGetter, currentMarketId, currentMarketPositionData, maybeDrawPositionLine]);
 
+  // Cache for order modification that stores the new orders that are submitted but not yet placed
+  const pendingOrderAdjustmentsRef = useRef<{
+    [clientId: string]: { orderPayload: HumanReadablePlaceOrderPayload; oldOrderId: string };
+  }>({});
+
+  const removePendingOrderAdjustment = (clientId: number) => {
+    const { [clientId]: removed, ...withoutOrderId } = pendingOrderAdjustmentsRef.current;
+    pendingOrderAdjustmentsRef.current = withoutOrderId;
+  };
+
+  const addPendingOrderAdjustment = (
+    orderPayload: HumanReadablePlaceOrderPayload,
+    oldOrderId: string
+  ) => {
+    pendingOrderAdjustmentsRef.current = {
+      ...pendingOrderAdjustmentsRef.current,
+      [orderPayload.clientId]: { orderPayload, oldOrderId },
+    };
+  };
+
+  const dispatch = useAppDispatch();
```

Comment:
super duper nit, can we move this to the top of the func (I think that's generally the pattern in our codebase, just makes it easier to skim over to see if it exists)